Cannot compile a typescript application with latest release of profiler.

When I try to compile a typescript module with the profiler added, I get this error:
```
node_modules/@google-cloud/profiler/out/src/index.d.ts:16:34 - error TS7016: Could not find a declaration file for module 'console-log-level'. '/usr/local/google/home/nolanmar/ts-test/test-ts/node_modules/console-log-level/index.js' implicitly has an 'any' type.
  Try `npm install @types/console-log-level` if it exists or add a new declaration (.d.ts) file containing `declare module 'console-log-level';`

16 import * as consoleLogLevel from 'console-log-level';
                                    ~~~~~~~~~~~~~~~~~~~

node_modules/@google-cloud/profiler/out/src/index.d.ts:17:24 - error TS7016: Could not find a declaration file for module 'semver'. '/usr/local/google/home/nolanmar/ts-test/test-ts/node_modules/semver/semver.js' implicitly has an 'any' type.
  Try `npm install @types/semver` if it exists or add a new declaration (.d.ts) file containing `declare module 'semver';`

17 import { SemVer } from 'semver';
                          ~~~~~~~~

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! test-ts@1.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the test-ts@1.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /usr/local/google/home/nolanmar/.npm/_logs/2018-09-28T16_55_15_799Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -27,6 +27,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@google-cloud/common": "^0.25.0",
+    "@types/console-log-level": "^1.4.0",
+    "@types/semver": "^5.5.0",
     "bindings": "^1.2.1",
     "console-log-level": "^1.4.0",
     "delay": "^4.0.1",
@@ -41,7 +43,6 @@
     "teeny-request": "^3.3.0"
   },
   "devDependencies": {
-    "@types/console-log-level": "^1.4.0",
     "@types/extend": "^3.0.0",
     "@types/long": "^4.0.0",
     "@types/mocha": "^5.0.0",
@@ -50,7 +51,6 @@
     "@types/pify": "^3.0.0",
     "@types/pretty-ms": "^3.0.0",
     "@types/request": "^2.47.1",
-    "@types/semver": "^5.5.0",
     "@types/sinon": "^5.0.1",
     "codecov": "^3.0.0",
     "deep-copy": "^1.4.2",

--- a/testing/busybench/package.json
+++ b/testing/busybench/package.json
@@ -1,0 +1,27 @@
+{
+  "name": "",
+  "version": "0.0.0",
+  "description": "",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src"
+  ],
+  "license": "Apache-2.0",
+  "keywords": [],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
+  },
+  "devDependencies": {
+    "gts": "^0.8.0",
+    "typescript": "~2.8.0"
+  },
+  "dependencies": {}
+}

--- a/testing/busybench/src/busybench.ts
+++ b/testing/busybench/src/busybench.ts
@@ -13,15 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import * as profiler from '@google-cloud/profiler'
+profiler.start({
+  logLevel: 5,
+})
 
-const startTime = Date.now();
-const testArr = [];
+const startTime: number = Date.now();
+const testArr: Array<Array<number>> = [];
 
 /**
  * Fills several arrays, then calls itself with setImmediate.
  * It continues to do this until durationSeconds after the startTime.
  */
-function busyLoop(durationSeconds) {
+function busyLoop(durationSeconds: number){
   for (let i = 0; i < testArr.length; i++) {
     for (let j = 0; j < testArr[i].length; j++) {
       testArr[i][j] = Math.sqrt(j * testArr[i][j]);
@@ -32,13 +36,13 @@ function busyLoop(durationSeconds) {
   }
 }
 
-function benchmark(durationSeconds) {
+function benchmark(durationSeconds: number) {
   // Allocate 16 MiB in 64 KiB chunks.
   for (let i = 0; i < 16*16; i++) {
-    testArr[i] = new Array(64 * 1024);
+    testArr[i] = new Array<number>(64 * 1024);
   }
-  busyLoop(durationSeconds)
+  busyLoop(durationSeconds);
 }
 
-const durationSeconds = process.argv.length > 2 ? process.argv[2] : 600; 
+const durationSeconds = Number(process.argv.length > 2 ? process.argv[2] : 600); 
 setTimeout(()=>benchmark(durationSeconds), 1000);

--- a/testing/busybench/tsconfig.json
+++ b/testing/busybench/tsconfig.json
@@ -1,0 +1,13 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build"
+  },
+  "include": [
+    "src/*.ts",
+    "src/**/*.ts",
+    "test/*.ts",
+    "test/**/*.ts"
+  ]
+}

--- a/testing/integration_test.go
+++ b/testing/integration_test.go
@@ -92,13 +92,15 @@ PROFILER="$HOME/cloud-profiler-nodejs/google-cloud-profiler-$VERSION.tgz"
 
 TESTDIR="$HOME/test"
 mkdir -p "$TESTDIR"
-cp "testing/busybench.js" "$TESTDIR"
-cd "$TESTDIR"
+cp -r "testing/busybench" "$TESTDIR"
+cd "$TESTDIR/busybench"
 
 retry npm install "$PROFILER" >/dev/null
+retry npm install
+npm run compile
 
 # Run benchmark with agent
-GCLOUD_PROFILER_LOGLEVEL=5 GAE_SERVICE={{.Service}} node --trace-warnings --require @google-cloud/profiler busybench.js 600
+GCLOUD_PROFILER_LOGLEVEL=5 GAE_SERVICE={{.Service}} node --trace-warnings --require @google-cloud/profiler build/src/busybench.js 600
 
 # Indicate to test that script has finished running
 echo "busybench finished profiling"

